graphql/nearText.js

```diff
@@ -111,37 +121,59 @@ export default class GraphQLNearText {
       throw new Error("nearText filter: moveTo must be object");
     }
 
-    if (!Array.isArray(target.concepts)) {
+    if (!Array.isArray(target.concepts) && !Array.isArray(target.objects)) {
+      throw new Error("nearText filter: moveTo.concepts or moveTo.objects must be present");
+    }
+
+    if (target.concepts && !Array.isArray(target.concepts)) {
       throw new Error("nearText filter: moveTo.concepts must be an array");
     }
 
+    if (target.objects && !Array.isArray(target.objects)) {
+      throw new Error("nearText filter: moveTo.objects must be an array");
+    }
+
     if (target.force && typeof target.force != "number") {
       throw new Error("nearText filter: moveTo.force must be a number");
     }
 
     this.moveTo = true;
     this.moveToConcepts = target.concepts;
     this.moveToForce = target.force;
+    if (target.objects) {
+      this.moveToObjects = this.parseMoveObjects("moveTo", target.objects);
+    }
   }
 
   parseMoveAwayFrom(target) {
     if (typeof target !== "object") {
       throw new Error("nearText filter: moveAwayFrom must be object");
     }
 
-    if (!Array.isArray(target.concepts)) {
+    if (!Array.isArray(target.concepts) && !Array.isArray(target.objects)) {
```

Comment:
should be `!target.concepts && !target.objects` to reflect error message. Array type is checked later on.